src/main/java/frc/robot/maps/subsystems/ArmRotateMap.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.maps.subsystems;
+
+import org.checkerframework.checker.units.qual.degrees;
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+
+public class ArmRotateMap {
+    public SmartMotorController motor;
+
+    public ArmRotateMap() {
+        this.motor = new SmartMotorController();
+    }
+
+    public ArmRotateMap(SmartMotorController motor) {
+        this.motor = motor;
+    }
+
+    public void updateData(Data data) {
+        motor.set(data.setPoint);
+        data.degrees = motor.getEncoder().getDistance();
+        data.velocityMPS = motor.getEncoder().getRate();
```

Comment:
The unit here will be related to the unit of `getDistance()` so DegreesPerSecond.